plugins/dota.py

```diff
@@ -0,0 +1,175 @@
+import logging
+log = logging.getLogger(__name__)
+
+import os
+import sqlite3
+import dota2api
+import datetime
+import botologist.plugin
+from time import sleep
+
+class DotaPlugin(botologist.plugin.Plugin):
+    def __init__(self, bot, channel):
+        super().__init__(bot, channel)
+        api_key = self.bot.config.get('dota2_apikey')
+        self.api = dota2api.Initialise(api_key)
+        db = os.path.join(self.bot.storage_dir, 'dota2.db')
+        if not os.path.isfile(db):
+            open(db, "w+")
+        self.conn = sqlite3.connect(db, check_same_thread=False)
```

Comment:
A plugin gets initialised multiple times - once for each channel. Maybe this should be a class property instead of an instance property?